addons/stock_account/wizard/stock_valuation_layer_revaluation.py

```diff
@@ -39,7 +39,7 @@ def default_get(self, default_fields):
     new_value_by_qty = fields.Monetary("New value by quantity", compute='_compute_new_value')
     reason = fields.Char("Reason", help="Reason of the revaluation")
 
-    account_journal_id = fields.Many2one('account.journal', "Journal", required=True, check_company=True)
+    account_journal_id = fields.Many2one('account.journal', "Journal", check_company=True)
```

Comment:
Don't forget to put the required condition in the views if the `property_valuation == 'real_time'` (via attrs)